src/ad-ui.js

```diff
@@ -438,9 +438,9 @@ AdUi.prototype.onNonLinearAdLoad = function() {
  * @param {number} height The post-resize height of the player.
  */
 AdUi.prototype.onPlayerResize = function(width, height) {
-  if (this.adsManager) {
-    this.adsManagerDimensions.width = width;
-    this.adsManagerDimensions.height = height;
+  if (this.controller.sdkImpl.adsManager) {
+    this.controller.sdkImpl.adsManagerDimensions.width = width;
+    this.controller.sdkImpl.adsManagerDimensions.height = height;
     /* global google */
     /* eslint no-undef: 'error' */
     this.adsManager.resize(width, height, google.ima.ViewMode.NORMAL);
```

Comment:
This call will still fail since AdUi doesn't own adsManager, sdkImpl does. I think to fix this we actually want to move this whole method to SdkImpl, and in controller.js instead of calling adUi.onPlayerResize() we want to call sdkImpl.onPlayerResize().

Reply:
Sorry, that's a blatant oversight on line 446 that should read: 
```js
this.controller.sdkImpl.adsManager.resize(width, height, google.ima.ViewMode.NORMAL);
```

However, point taken. Will refactor into `sdkImpl.onPlayerResize()`